.travis.yml

```diff
@@ -29,7 +24,7 @@ env:
 
 before_install:
   - sudo apt-get update -qq
-  - sudo apt-get install -y --force-yes python-virtualenv jq liberasurecode-dev libssl-dev libattr1-dev libleveldb1 libleveldb-dev libzookeeper-mt-dev golang-${GO_VERSION}
+  - sudo apt-get install -y --force-yes python-virtualenv jq liberasurecode-dev libssl-dev libattr1-dev libleveldb-dev libzookeeper-mt-dev
```

Comment:
We could use `packages` to install these packages: https://github.com/open-io/oio-sds/blob/master/.travis.yml#L11

Reply:
The idea should to extract all install stuff in a dedicated script in `oio-sds` (it will be useful for other CIs (Jenkins, CircleCI, ...) or others projects like `oio-swift` (Work In Progress on my side)